package.json

```diff
@@ -19,7 +19,7 @@
     "node": ">=0.8"
   },
   "dependencies": {
-    "backbone": "1.1.2",
+    "backbone": "1.2.3",
```

Comment:
Although noticed that Bower is not locked... Pretty sure we should just drop Bower in this next release anyways.